src/app/pages/health-declaration/__tests__/health-declaration.page.spec.ts

```diff
@@ -1,26 +1,26 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { IonicModule, NavController } from '@ionic/angular';
-
 import { NavMock } from '@mocks/angular-mocks/nav-mock';
 import { RouteByCategoryProvider } from '@providers/route-by-category/route-by-category';
 import { RouteByCategoryProviderMock } from '@providers/route-by-category/__mocks__/route-by-category.mock';
-import { RekeyReasonCatBEPage } from '../rekey-reason.cat-be.page';
 
-describe('RekeyReasonCatBEPage', () => {
-  let component: RekeyReasonCatBEPage;
-  let fixture: ComponentFixture<RekeyReasonCatBEPage>;
+import { HealthDeclarationPage } from '../health-declaration.page';
+
+describe('HealthDeclarationPage', () => {
+  let component: HealthDeclarationPage;
+  let fixture: ComponentFixture<HealthDeclarationPage>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [RekeyReasonCatBEPage],
+      declarations: [HealthDeclarationPage],
       imports: [IonicModule.forRoot()],
       providers: [
         { provide: NavController, useClass: NavMock },
```

Comment:
whats navcontroller being provided/mocked for? most navcontroller stuff replaced by router